downstream/modules/platform/con-controller-resources.adoc

```diff
@@ -0,0 +1,11 @@
+[id="con-controller-resources"]
+
+= Resources Menu
+
+The *Resources* menu provides access to the various components of {ControllerName}.
```

Comment:
"The *Resources* menu provides access to the following components of {ControllerName}:"